browser-sign-in/okta.properties

```diff
@@ -0,0 +1,4 @@
+discoveryUrl=${CLI_OKTA_ORG_URL}oauth2/default/.well-known/openid-configuration
```

Comment:
Should we update these to remove CLI references? Here's what I did in okta-mobile-kotlin: https://github.com/okta/okta-mobile-kotlin/blob/master/okta.properties

Reply:
It's now matching okta-mobile-kotlin